pkg/operator/cluster/ceph/osd/osd.go

```diff
@@ -311,6 +311,11 @@ func makeOrchestrationStatusMap(clientset kubernetes.Interface, namespace string
 			},
 			Data: make(map[string]string),
 		}
+		// the owner ref will be set the first time when the cluster is created, but if it is created later by the osd daemon
+		// we skip setting the owner since we are in an unexpected state of the configmap not already existing.
```

Comment:
i believe that the `UpdateOrchestrationStatusMap` also calling `makeOrchestrationStatusMap` was probably to help for unit tests, so this decision seems like an OK one here.